app/views/repositories/_revisions.html.erb

```diff
@@ -134,8 +135,9 @@ See COPYRIGHT and LICENSE files for more details.
                                      (line_num == 2),
                                      id: "cbto-#{line_num}" %>
                   <% csp_onclick(
-                       "if (jQuery('#cb-#{line_num}').attr('checked')) {jQuery('#cb-#{line_num - 1}').attr('checked', true)}",
-                       "cbto-#{line_num}"
+                       "if (document.getElementById('cb-#{line_num}')?.checked) {document.getElementById('cb-#{line_num - 1}').checked = true;}",
```

Comment:
This is the original behavior - but not necessarily the most sensible behavior. It looks like it was broken some years ago: https://github.com/opf/openproject/commit/28ff6e0fc93bbc010dee7403be0f09f0415b1e52